src/applications/personalization/preferences/components/PreferenceItem.jsx

```diff
@@ -4,21 +4,50 @@ import { Link } from 'react-router';
 import AdditionalInfo from '@department-of-veterans-affairs/formation/AdditionalInfo';
 import AlertBox from '@department-of-veterans-affairs/formation/AlertBox';
 
+const CallToAction = ({ cta }) => {
+  const { description, link, text } = cta;
+  return (
+    <div>
+      {description && description}
+      {link &&
+        text && (
+          <Link className="usa-button" to={link}>
+            {text}
+          </Link>
+        )}
+    </div>
+  );
+};
```

Comment:
Is it enough to just do `{ description }` here, or does that render differently?

Reply:
Good call, updated.